mozilla-release/mobile/android/app/src/main/res/layout/tab_menu_strip.xml

```diff
@@ -3,12 +3,13 @@
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <!--Cliqz Start-->
 <ImageView xmlns:android="http://schemas.android.com/apk/res/android"
-        android:id="@+id/icon"
-        android:layout_width="0dp"
-        android:layout_weight="1"
-        android:layout_height="36dp"
-        android:focusable="true"
-        android:padding="5dp"
-        android:layout_gravity="center"
-        android:scaleType="fitCenter" />
+    xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
Why are you changing the indentation? This is how you make yourself sure to have a conflict next time we merge a new Firefox.